docs/run/execution-modes.mdx

```diff
@@ -0,0 +1,46 @@
+
+<span id="best-practices"></span>
+## Best practices
+
+To ensure the most efficient use of the execution modes, the following practices are recommended:
+
+- Always close your session, either by using a context manager or by specifying `session.close()`.
+- There is a fixed overhead associated with running a job. In general, if your each of your job uses less than one minute of QPU time, consider combining several into one larger job (this applies to all execution modes). "QPU time" refers to time spent by the QPU complex to process your job.
+
+    A job's QPU time is listed in the **Usage** column on the IQP [Jobs](https://quantum.ibm.com/jobs) page, or you can  query it by using  `qiskit-ibm-runtime` with this command `job.metrics()["usage"]["quantum_seconds"]`
+
+- If each of your jobs consume more than one minute of QPU time, or if combining jobs is not practical, you can still run multiple jobs in parallel. Every job goes through both classical and quantum processing. While a QPU can process only one job at a time, up to five classical jobs can be processed in parallel. You can take advantage of this by submitting multiple jobs in [batch](run-jobs-batch#partition) or [session](run-jobs-in-session#two-vqe) execution mode.
+
+The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, running smaller jobs in parallel might be more cost effective because of reduced wall clock time.
+
+## Sessions versus batch usage
+
+Usage is a measurement of the amount of time the system is locked for your workload.
+
+* Session usage is the entire active session time.
+* Batch usage is the amount of time all jobs spend on the system.
+* Single job usage is the quantum time they use in processing.
+
+![This image shows multiple sets of jobs.  One set is being run in sessions mode and the other is being run in batch mode.  For sessions mode, between each job is the interactive TTL (time to live).  The active window starts when the first job starts and ends after the last job is completed. After the final job of the first set of jobs completes, the active window ends and the session is paused (but not closed).  Another set of jobs then starts and jobs continue in a similar manner. The system is reserved for your use during the entire session.  For batch mode, the classical computation part of each job happens simultaneously, then all jobs are sent to the system.  The system is locked for your use from the time the first job reaches the system until the last job is done processing on the system.  There is no gap between jobs where the system is idle.](/images/run/execution-modes/SessionVsBatch.svg 'Sessions compared to batch')
```

Comment:
Can we add code telling people how to retrieve the usage time for a batch or session? This can also go into FAQ if you'd prefer keeping the page simple. Something like

"Using qiskit-ibm-runtime 0.23 or above, you can use `session.details()["usage_time"]` or `batch.details()["usage_time"]` to find the usage time for your Session or Batch, respectively. 

If you are calling the REST API directly, the usage time is the `elapsed_time` returned via GET /sessions/{id} endpoint, for both Session and Batch. "